vumi_twilio_api/twilio_api.py

```diff
@@ -149,10 +149,25 @@ def _handle_connected_call(
         self.session_manager.save_session(session_id, session)
         twiml = yield self._get_twiml_from_client(session)
         for verb in twiml:
-            self._handle_twiml_verb(verb)
+            yield self._handle_twiml_verb(verb, session)
 
-    def _handle_twiml_verb(self, verb):
-        pass
+    @inlineCallbacks
+    def _handle_twiml_verb(self, verb, session):
+        if not verb:
+            return
```

Comment:
I think this happens in one of the tests, where we patch the `parse_Say` function, and it doesn't return anything. We can either change it to return something, or keep this in.